doc/source/reference/c-api.array.rst

```diff
@@ -53,6 +68,19 @@ sub-types).
     array then be sure you understand how to access the data in the
     array to avoid memory and/or alignment problems.
 
+.. c:function:: npy_intp *PyArray_DIMS(PyArrayObject *arr)
+
+    Returns a pointer to the dimensions/shape of the array. The
+    number of elements matches the number of dimensions
+    of the array. Can return ``NULL`` for 0-dimensional arrays.
+
+.. c:function:: npy_intp *PyArray_SHAPE(PyArrayObject *arr)
+
+    .. versionadded:: 1.7
+
+    A synonym for PyArray_DIMS, named to be consistent with the
+    'shape' usage within Python.
```

Comment:
do you need `ndarary.shape` (single backticks) to turn it into a link?

Reply:
Single backticks just make the test italic. For including the link I did something like the recent change :attr:`shape <ndarray.shape>` as was done in the other document `arrays.ndarray.rst` but for some reason, that simply made the text bold but did not link it with the attributes' documentation. Doing `:ref:` doesn't even make the text bold, leave rendering as a link.

Reply:
this works for me:
```
`shape <numpy.ndarray.shape>`
```

You could avoid the `numpy` prefix by adding `.. currentmodule:: numpy` before this